components/edit-team-form.js

```diff
@@ -1,9 +1,31 @@
 import React from 'react'
 import { Formik, Field, Form } from 'formik'
+import urlRegex from 'url-regex'
```

Comment:
Should validation also happen at the API level?

Reply:
yeah it probably should